app/src/main/java/com/codesoom/assignment/AppHttpHandler.java

```diff
@@ -0,0 +1,131 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.enums.HttpMethodType;
+import com.codesoom.assignment.error.ClientError;
+import com.codesoom.assignment.models.Task;
+import com.codesoom.assignment.service.TaskService;
+import com.codesoom.assignment.utils.JsonUtil;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class AppHttpHandler implements HttpHandler {
+    private final TaskService taskService = new TaskService();
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        HttpMethodType method = HttpMethodType.valueOf(exchange.getRequestMethod());
+        String[] pathArr = exchange.getRequestURI().getPath().split("/");
+        String path = pathArr[1];
+        Long userId = getUserId(pathArr);
+        String requestBody = new BufferedReader(new InputStreamReader(exchange.getRequestBody()))
+                .lines()
+                .collect(Collectors.joining("\n"));
+
+        if (!path.equals("tasks")) {
```

Comment:
이렇게 하면 `path`가 null이어도 NPE가 던져지지 않을 거에요.
```suggestion
        if (!"tasks".equals(path)) {
```

Reply:
null 위험요소를 뒤로 보내서 NPE를 피하는 방법이군요!
감사합니다 :)
해당 부분은 enum 클래스인 ContextContainer에서 stream orElse로 수정해보았습니다! (350ab66)